python/python/ert_gui/plottery/plot_data_gatherer.py

```diff
@@ -72,11 +72,16 @@ def gatherSummaryData(ert, case, key):
         if not data.empty:
             data = data.reset_index()
 
-            if True in data.duplicated():
```

Comment:
I think the correct syntax is `True in data.duplicated().values`or even better `data.duplicated().any()`

Reply:
Did not see Andrea's comment. We should us a form of `any`. I think the current code looks for `True` in the indices.

Reply:
yes, I have just checked, doing `'whatever' in data.duplicated()` would check `whatever` against the labels, not against the boolean flag